roles/network_plugin/kube-ovn/templates/cni-kube-ovn.yml.j2

```diff
@@ -608,3 +608,17 @@ spec:
   ports:
     - port: 10665
       name: metrics
+---
```

Comment:
The OVN Interconnect may not be used by default.  shoud we add a var to "enable" it, like:

if ovn-ic-config-enabled
....
endif